scripts/txt_replace.py

```diff
@@ -65,13 +72,17 @@ def fix_dockerfile(version_tuple):
         replace(filename, k, v)
 
 if __name__ == "__main__":
-    nargs = len(sys.argv)
-    if nargs != 4:
-        print("usage: %s VERSION PREVIOUS_VERSION IMAGE_SHA" % path.basename(sys.argv[0]))
-        exit()
+    parser = argparse.ArgumentParser(description="Script for updating the appropriate fields of the CSV",
+                                 formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+    parser.add_argument("-n", "--namespace", help="namespace used for testing")
```

Comment:
This should default to `""` to avoid inserting `'namespace: None'` in `fix_csv()`.
```suggestion
    parser.add_argument("-n", "--namespace", help="namespace used for testing", default="")
```

Reply:
done.